application/src/main/java/org/togetherjava/tjbot/commands/basic/VcActivityCommand.java

```diff
@@ -37,6 +38,12 @@
 public final class VcActivityCommand extends SlashCommandAdapter {
     private static final Logger logger = LoggerFactory.getLogger(VcActivityCommand.class);
 
+    private static final int MIN_AGE_DURATION = 0;
+    private static final long MAX_AGE_DURATION = TimeUnit.DAYS.toSeconds(7);
```

Comment:
The constants should have the unit in their name to prevent confusion. `MIN/MAX_AGE_DURATION_SECONDS`

Reply:
Fixed